alerts/apps_alerts.libsonnet

```diff
@@ -143,7 +143,7 @@
           {
             expr: |||
               (
-                max without (revision) (
+                max by(namespace, statefulset) (
```

Comment:
Sample `kube_statefulset_status_current_revision` expositions (out-of-the-box, from KSM).

```yaml
# HELP kube_statefulset_status_current_revision [STABLE] Indicates the version of the StatefulSet used to generate Pods in the sequence [0,currentReplicas).
# TYPE kube_statefulset_status_current_revision gauge
kube_statefulset_status_current_revision{namespace="default",statefulset="prometheus-prometheus-stack-kube-prom-prometheus",revision="prometheus-prometheus-stack-kube-prom-prometheus-7485456df4"} 1
kube_statefulset_status_current_revision{namespace="default",statefulset="alertmanager-prometheus-stack-kube-prom-alertmanager",revision="alertmanager-prometheus-stack-kube-prom-alertmanager-f54678bb9"} 1
kube_statefulset_status_current_revision{namespace="default",statefulset="nginx",revision="nginx-8c4dcc797"} 1
```